### ISSUE TITLE
Bug with datetime values in `timezone.utc`

### ISSUE THREAD

**Issue by ajmarks** [has2k1/mizani#30]
The changes at https://github.com/has2k1/mizani/commit/bee89773c11283df63a845feeb809982c0c8578d cause a `ValueError` (at [`mizani/utils.py:360`](https://github.com/has2k1/mizani/commit/bee89773c11283df63a845feeb809982c0c8578d#diff-a9554838ef0153a96280c36ceebcc5143a098328f4e0e936dcd705b07ee890f9R360)) when with datetimes using `timezone.utc`.  Repro: 

```python
from datetime import datetime, timezone
from mizani.utils import get_timezone

dt = datetime.now(tz=timezone.utc)
get_timezone([dt])
```

**Comment by repnant** [comment]
Adding a fuller error message, as this was an extremly frustratingly non-google-able-bug, and it took me way too long too figure out that I needed to demote mizani to 0.8.1 when using plotnine 

```
... mizani/utils.py", line 360, in get_timezone
    raise ValueError(f"Unrecognised timezone class {info.__class__}")
ValueError: Unrecognised timezone class <class 'datetime.timezone'>
```

**Comment by has2k1** [comment]
Fix has been released in mizani-0.9.2. Now on pypi, and will be available on conda later.
